### PR TITLE
2020.04+imx 5.4.70 2.3.0 fio

### PR DESCRIPTION
**Pull request by vanmaegima**
Rebasing FIO patches on top of https://source.codeaurora.org/external/imx/uboot-imx/log/?h=imx_v2020.04_5.4.70_2.3.0 + https://git.toradex.com/cgit/u-boot-toradex.git/log/?h=toradex_imx_v2020.04_5.4.70_2.3.0

Tested on imx8mmevk, imx6ullevk and apalis-imx6.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,18 +1,26 @@
 # SPDX-License-Identifier: GPL-2.0+
```
Comment (angolini) [inline .gitlab-ci.yml:1]: I don't know if changing `.gitlab-ci.yml` can hurt anything. But maybe this commit is not exactly needed also.

**Review by angolini — APPROVED** [review]
I think that's it, Vanessa

**Review by vanmaegima — COMMENTED** [review]
@ricardosalveti @mike-scott @Tim-Anderson first patch to fix the imx8mm defconfig renaming issue based on Toradex approach, still need a patch to dynamically set the correct fdt_file

https://github.com/foundriesio/u-boot/pull/51/commits/19965d71a302e2c28cccb755d61bb4ef38ee0bd1

**Comment by vanmaegima** [comment]
@ricardosalveti @mike-scott added patch for dynamically setting the fdt_file for the imx8mm evk board variants 0b33f54abdb930166f71110042dd4fe59087401d

File: .gitlab-ci.yml (hunk 1 above)
Comment (igoropaniuk) [reply .gitlab-ci.yml:1]: it's not even used in the mainline U-Boot now, so yeah it can be just dropped

**Comment by vanmaegima** [comment]
added 2fd3afb [FIO hack] imx: fiohab: ignore known HAB events

**Comment by vanmaegima** [comment]
PR rebased on top of last changes by NXP and Toradex.
Added needed patches for FIO compatibility.
Tested on imx6ullevk and imx8mmevk (needs linux kernel update for new dtb).

**Review by igoropaniuk — APPROVED** [review]

**Review by ricardosalveti — APPROVED** [review]
Looks great, thanks for the rebase and sorry it took so long to merge.

Now for testing it all together with hardknott :-)

**Pull request merged**